src/main/resources/manifeststore.yaml

```diff
@@ -104,4 +104,3 @@ paths:
          schema:
            type: string
          description: The manifest ID
-  
```

Comment:
I can never remember if this means the line has been removed, but we do want a blank line at the end of our files. Checkstyle checks a bunch of different formats but probably doesn't check YAML files, yes.